lib/castle/default_context.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Castle
+  class DefaultContext
+    def initialize(headers, ip)
+      @headers = headers
+      @ip = ip
+    end
+
+    # rubocop:disable Metrics/MethodLength
+    def call
+      context = {
+        active: true,
+        origin: Castle.config.source_header,
+        request: {
+          headers: @headers || {}
+        },
+        ip: @ip,
+        library: {
+          name: 'castle-rb',
+          version: Castle::VERSION
+        }
+      }
+      context[:request][:remoteAddress] = @headers['Remote-Addr'] if @headers['Remote-Addr']
```

Comment:
@bartes feel free to change it directly. I will update the doc later today, too so I can send it over to the Java SDK developer